configs/stable-diffusion-pretraining/gce.yaml

```diff
@@ -0,0 +1,20 @@
+head_node_type:
+  name: head
+  instance_type: n2-standard-8
+  resources:
+    cpu: 0
+auto_select_worker_config: false
```

Comment:
Why can this not use serverless?

Reply:
My desired goal is a single GPU, with no autoscaling, a very simple set up to ensure the smoothest UX for anyone trying out the template.

Reply:
Can't you just enforce a single GPU via the application?

Reply:
I see - I am primarily optimizing for "time to run notebook"

but I see your point in not consuming GPU units and I am willing to try it your way - perhaps will make the change and then test it out - if it is still a reasonable experience will keep as is, otherwise will escalate

Reply:
The upscaling speed will be much faster in production, where we are buffering warm A10G's. If you are using the slim image as your base, upscaling an A10G should be on the order of < 20 seconds.

Reply:
ok nice - yes I am hoping to make use of the slim image. Sounds promising

Reply:
@shomilj what about `allow-cross-zone-autoscaling` should I be setting this to true or false? what considerations should I keep in mind  here? - I am thinking cross-zone autoscaling entails a delay so perhaps best to keep it false ?